roles/servicetelemetry/templates/manifest_snmp_traps_deploy.j2

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: '{{ meta.name }}-snmp-webhook'
+  namespace: '{{ meta.namespace }}'
+spec:
+  selector:
+    matchLabels:
+      app: '{{ meta.name }}-snmp-webhook'
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: '{{ meta.name }}-snmp-webhook'
+    spec:
+      containers:
+        - name: 'prometheus-webhook-snmp'
+          image: 'quay.io/mrunge/prometheus-webhook-snmp:latest'
```

Comment:
We will still need to get this code into the github.com/infrawatch organization and then setup builds so that the image can be pulled from quay.io/infrawatch. This blocks merge into main branch.

Reply:
What has to be done? How?

Reply:
Get the code into github, then we need to create a corresponding repo on quay.io/infrawatch and hook up the build triggers to create new images on merge to master.

Reply:
If you get the github repo created and the code imported, I can do the rest since it won't take me very long.

Reply:
https://github.com/mrunge/prometheus-webhook-snmp